File: trunk/ShadowFrameworkLite20/src/shadow/image/SFTextureDataToken.java
package shadow.image;

public class SFTextureDataToken extends SFPipelineTexture{

	public SFTextureDataToken(int width, int height, SFImageFormat format,
			Filter filters, WrapMode wrapS, WrapMode wrapT) {
		super(width, height, format, filters, wrapS, wrapT);
	}

	@Override
	public void apply(int textureLevel) {
		//Do nothing
	}
	
}

File: ShadowFrameworkJS/src/shadow/image/SFTextureDataToken.js



function SFTextureDataToken(width, height, format, filters,  wrapS, wrapT){
	this.filters = filters;
	this.WrapS = wrapS;
	this.WrapT = wrapT;
	this.width=width;
	this.height=height;
	this.format=format;
}

inherit(SFTextureDataToken,SFPipelineTexture);

SFTextureDataToken.prototype["apply"]=new function(textureLevel){
	//Do nothing
};